content/api/resources/operations/object/create-claimable-balance.mdx

```diff
@@ -43,7 +43,14 @@ Creates a new claimable balance.
           - The value is also a predicate. This clause of the condition is satisfied if the value is _not_ satisfied.
         - absBefore
           - string (optional)
-          - An ISO 8601 formatted string representing a deadline for when the claimable balance can be claimed. If the balance is claimed before the date then this clause of the condition is satisfied.
+          - A customized ISO 8601 formatted string representing a deadline for when the claimable balance can be claimed. If the balance is claimed before the date then this clause of the condition is satisfied. The format of this date string is a custom extension on top of ISO 8601 format. It allows for years to be outside the 0000-9999 range. The dates are derived from a unix epoch value in range of signed 64 bit integer. This means the date expresses a much larger calendar range of 292277026596 years into future and -292471206707 years back in past. This custom extension format will add a `'+'` prefix on values that go beyond year 9999 into the future and for years that are prior to year 0(B.C per Gregorian calendar) it will add prefix of `'-'`.  
```

Comment:
@Shaptic , I wanted to confirm on the change here, is the intent to highlight the API history resource model(json) for ClaimableBalance or is it meant to show the model for creating a ClaimableBalance operation in a transaction?  I might need to tweak this if it's for the latter purpose, since that model only has absBefore and it's a uint64 value not a ISO 8601 date string, encoded in XDR at that point.

Reply:
It's the former: this is the JSON object that gets returned from Horizon after you've submitted a CreateClaimableBalanceOp within a transaction to the network (see [here](https://developers.stellar.org/api/resources/operations/object/create-claimable-balance/)). The latter is covered by [this page](https://developers.stellar.org/docs/start/list-of-operations/#create-claimable-balance).